### PR TITLE
word embedding tutorial review

### PR DESCRIPTION
**Pull request by Tobai24**
Hello  @Lexo7  👋, thanks for the great work on this!

Just a couple of small notes:

* **Embedding method note**: You’re currently using the `[CLS]` token to represent BERT sentence embeddings. While this is common, many recent works suggest that mean pooling over all token embeddings tends to yield better results for semantic similarity tasks. It might be worth testing this alternative.

* **Evaluation suggestion**: To make the comparison between GloVe and BERT embeddings more robust, you could compute a correlation metric (like Pearson or Spearman) against the gold similarity scores from the STS dataset. This would complement the visual plot with a quantitative measure.

* **Missing imports**: (I've included the comment in the notebook)

  ```python
  import numpy as np
  from sklearn.metrics.pairwise import cosine_similarity
  ```

Let me know what you think happy to chat more or help clarify if needed!

### CODE REVIEW

**Comment by Lexo7** [comment]
yes please. I will make the necessary changes and revert
